README.md

```diff
@@ -359,6 +359,31 @@ View Kubescape scan results directly in [Lens IDE](https://k8slens.dev/) using k
 
 </details>
 
+
+## Build on pre-configured killercoda's ubuntu playground
+
+* [Pre-configured Killercoda's Ubuntu Playground](https://killercoda.com/suhas-gumma/scenario/kubescape-build-for-development)
+
+<details><summary> Pre-programmed actions executed by the playground </summary>
+
+
+* Clone the official GitHub repository of `Kubescape`.
+* Install cmake.
```

Comment:
I think we should link here to the [Build on Linux](https://github.com/kubescape/kubescape#build-on-linuxmacos)

Reply:
Hi @dwertent , I have made the changes you suggested.